packages/react-composites/src/composites/CallComposite/components/buttons/People.tsx

```diff
@@ -27,22 +29,34 @@ export const People = (props: ControlBarButtonProps): JSX.Element => {
             background: theme.palette.neutralLight
           }
         },
-        props.styles ?? {},
         controlButtonBaseStyle
       ),
-    [props.styles, theme.palette.neutralLight]
+    [theme.palette.neutralLight]
+  );
+
+  const locale = useLocale();
+  // FIXME (?): Why is this using callWithChat strings?
```

Comment:
Thanks for the catch!
Submitted a PR to address this:
https://github.com/Azure/communication-ui-library/pull/2358